src/android/index.jsx

```diff
@@ -244,102 +282,27 @@ class Android extends Component {
           subtitle="Lower in the graph is better regardless if it is a score or execution time (read the Y label)"
         >
           <Grid container spacing={1}>
-            <Grid item xs={6}>
-              <PerfherderGraphContainer
-                timeDomain={timeDomain}
-                title="Speedometer"
-                series={[
-                  {
-                    label: 'Moto G5 (arm7)',
-                    filter: {
-                      and: [
-                        { missing: 'test' },
-                        {
-                          prefix: { platform: 'android-hw-g5-7-0-arm7-api-16' },
-                        },
-                        {
-                          eq: {
-                            framework: 10,
-                            repo: 'mozilla-central',
-                            suite: 'raptor-speedometer-geckoview',
-                          },
-                        },
-                      ],
-                    },
-                  },
-                  {
-                    label: 'Pixel 2 (ARM64)',
-                    filter: {
-                      and: [
-                        { missing: 'test' },
-                        {
-                          eq: {
-                            framework: 10,
-                            options: 'opt',
-                            platform: 'android-hw-p2-8-0-android-aarch64',
-                            repo: 'mozilla-central',
-                            suite: 'raptor-speedometer-geckoview',
-                          },
-                        },
-                      ],
-                    },
-                  },
-                ]}
-                missingDataInterval={10}
-              />
-            </Grid>
+            {speedometerChart}
             <Grid item xs={6}>
               <PowerSummary
                 key="power"
                 browser="geckoview"
                 suite="speedometer"
                 timeDomain={timeDomain}
+                platform="g5"
               />
             </Grid>
+
             <Grid item xs={6}>
-              <PerfherderGraphContainer
+              <PowerSummary
+                key="power"
+                browser="geckoview"
```

Comment:
You can remove the `browser` from the props and in `PowerSummary` component:
```
urls={{
  title: 'show details',
  url: URL({
	path: '/power/details',
	query: { browser, suite },
  }),
  icon: DetailsIcon,
}}
```
Building the url to `power/details` does not require the `browser`, so you can remove it's mentions as it's not useful anymore

Reply:
Hey @airimovici, I have removed the `browser` pop from `PoweSummary` component. 
Thanks for the heads up.